packages/aws-cdk/README.md

```diff
@@ -971,6 +971,22 @@ cdk gc --unstable=gc --action=delete-tagged --rollback-buffer-days=30
 
 This will delete assets that have been unused for >30 days, but will not tag additional assets.
 
+Here is a diagram that shows the algorithm of garbage collection:
+
+![Diagram of Garbage Collection algorithm](/packages/aws-cdk/images/garbage-collection.png)
```

Comment:
This image URL will not work outside GitHub; if we replicate the Markdown elsewhere, the relative link is very unlikely to point to the correct location. Make it an absolute GitHub link instead.